worker/draft.go

```diff
@@ -508,7 +507,13 @@ func (n *node) commitOrAbort(pkey string, delta *pb.OracleDelta) error {
 		if txn == nil {
 			return
 		}
+		retry := Config.MaxRetries
 		for err := txn.CommitToDisk(&writer, commit); err != nil; {
+			if retry <= 0 {
+				glog.Warningf("Max commit retry limit (%d) reached, ", Config.MaxRetries)
```

Comment:
Add a message saying unable to persist a commit to disk. Giving up now would result in a complete or partial loss of committed txn.